packages/steiger-plugin-fsd/src/forbidden-imports/index.spec.ts

```diff
@@ -12,7 +12,7 @@ vi.mock('tsconfck', async (importOriginal) => {
           compilerOptions: {
             baseUrl: '/src/',
             paths: {
-              '@/*': ['./*'],
+              '@/*': ['/users/user/project/src/*'],
```

Comment:
issue (non-blocking): this is slightly incorrect from the perspective of tsconfig semantics. When `baseUrl` is set, `paths` are computed relative to it. Therefore, it's a little strange when `baseUrl` and the alias resolve to completely different locations

suggestion: 
```suggestion
            baseUrl: '/users/user/project/src/',
            paths: {
              '@/*': ['./*'],
```

Reply:
Yes, you're completely right. I'm surprised that it hasn't broken anything